app/src/main/java/org/kinecosystem/kinit/model/earn/Question.kt

```diff
@@ -21,6 +23,13 @@ data class Question(val id: String? = null,
     }
 }
 
+data class QuizData(@SerializedName("answer_id")
+                    val answer_id: String,
+                    @SerializedName("explanation")
+                    val explanation: String,
+                    @SerializedName("reward")
+                    val reward: Int)
+
 fun Question.isValid(): Boolean {
     if (id.isNullOrBlank() || text.isNullOrBlank() || type.isNullOrBlank() || answers.orEmpty().isEmpty()) {
```

Comment:
No validation for quiz_data (if not `null`)?

Reply:
@berryve done